pkg/virt-api/rest/subresource.go

```diff
@@ -908,48 +908,26 @@ func (app *SubresourceAPIApp) FilesystemList(request *restful.Request, response
 }
 
 func generateVMVolumeRequestPatch(vm *v1.VirtualMachine, volumeRequest *v1.VirtualMachineVolumeRequest) (string, error) {
-	verb := "add"
-	if len(vm.Status.VolumeRequests) > 0 {
-		verb = "replace"
-	}
-
+	verb := getVerbStatus(vm)
```

Comment:
the verb doesn't really relate to a status, rather patch action.
`getPatchVerb` or `getPatchActionVerb` may be a more sutable name.

Reply:
Also, passing only  the VolumeRequests may be more descriptive